recipes/copulae/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "copulae" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4b6963c488ebec36fb91fb5cabd07c14a4d1c596df11c4865dd3b49a549157b8
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
```

Comment:
The `host` section only needs to contain the dependencies used when installing the package. You probably only need `python` and `pip`.

Reply:
Will give this a shot